crates/holochain_sqlite/tests/db_wrapper.rs

```diff
@@ -132,50 +135,6 @@ async fn reader_permits_are_limited() {
     assert_eq!(3, failed_count);
 }
 
-#[cfg(all(feature = "slow_tests", feature = "test_utils"))]
-#[tokio::test(flavor = "multi_thread")]
-async fn reader_permits_can_be_released() {
```

Comment:
I saw that you want to remove `conn_permit`, but haven't done so yet. Is it appropriate to remove this now?

Also can you explain more of the rationale for doing so? I believe the semaphore was intended to be used to provide some backpressure.

Reply:
I can't remove it without more changes because it's used by `holochain_cascade`. 

I'm not trying to remove the semaphores because they are doing a useful job, I'm just trying to make it harder to hold them for a long time because that is artificially limiting database throughput. So instead of being able to grab a permit, hold it, then link that to a database connection later the only option will be to do a database read/write and wait for a semaphore as part of that operation.

Until I move existing code over to use that method though I can't remove the `conn_permit` function